shared/src/types/ibc.rs

```diff
@@ -898,25 +903,42 @@ pub fn update_client(
     client_state: AnyClientState,
     headers: Vec<AnyHeader>,
 ) -> Result<(AnyClientState, AnyConsensusState)> {
-    let client = AnyClient::from_client_type(client_state.client_type());
-    let mut client_state = client_state;
-    let mut consensus_state = None;
-    for header in headers {
-        let (new_client_state, new_consensus_state) = client
-            .check_header_and_update_state(client_state, header.clone())
```

Comment:
I replaced `check_header_and_update_state` function with just updating states.
That's because `check_header_and_update_state` has been changed and we don't need to check the headers and states here. (IBC VP should check them)
I suppose that `ibc-rs` should provide a function that updates states without checking.